app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ReturnSavedController.scala

```diff
@@ -33,20 +33,20 @@ import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class ConfirmDraftReturnController @Inject() (
+class ReturnSavedController @Inject() (
```

Comment:
what was the reason for the name change?

Reply:
the first one sounded funny - what's actually happening is we're telling the user that the draft return has been saved. "Confirming" a draft return doesn't quite make sense

Reply:
could you rename this to `DraftReturnSavedController` pls?

Reply:
done